### DISCUSSION TITLE
Node のバージョン別にテストを実施したい

### DISCUSSION THREAD

**Discussion by koichimurakami** [mbc-net/mbc-cqrs-serverless#60] [Ideas]
Node.jp 18, 20, 22 等ユーザが使用するバージョンが異なることが想定されます。
MBC CQRS サーバーレス フレームワークでサポートする Node.js のバージョン別にUnitテストを実施してそれぞれのバージョンでテストがPassされる事を確認してからマージをするようにしたいです。

**Comment by baquan1708** [comment]
@koichimurakami cc: @Tuan-Anh-Nguyen-pionero 

I've just created PR: https://github.com/mbc-net/mbc-cqrs-serverless/pull/105. When you have a moment, could you please review it?
